src/KalmanFilter.java

```diff
@@ -0,0 +1,77 @@
+package sye8.utils;
+
+/**
+ * Simple implementation of the Kalman Filter for 1D data, without any dependencies
+ * Originally written in JavaScript by Wouter Bulten
+ * 
+ * Now rewritten into Java
+ * 
+ * @author yesifan
+ * 
+ * @see https://github.com/wouterbulten/kalmanjs
+ *
```

Comment:
Can you add the license of the project here? See for example the docblock on the js file.